services/ui-src/src/utils/forms/schemas.ts

```diff
@@ -16,6 +16,21 @@ export const number = () =>
     .required(error.REQUIRED_GENERIC);
 export const numberOptional = () => number().notRequired();
 
+// Mixed
+export const numberOrNA = () =>
+  numberSchema()
+    .transform(validateNA)
+    .required()
+    .nullable()
+    .typeError("Please input a Number OR 'N/A'");
+
+function validateNA(value: any, originalValue: string) {
+  if (typeof originalValue === "string" && originalValue === "N/A") {
+    return null;
+  }
+  return value;
+}
```

Comment:
Two thoughts:
  1. Not sure if we're going to be passing `null` here or `N/A`, but we'll figure this out.
  2. I think we might need to accept more than just `N/A` (for instance `N/A`, `n/a`, `Not applicable`, etc.). Maybe the `originalValue === "N/A"` check becomes a `.includes` check against an array of valid not-applicable-adjacent answers? Should prob build this way to future-proof.